src/common/utils/utils.ts

```diff
@@ -4,21 +4,6 @@ import Big from 'big.js';
 
 import { PARACHAIN_URL } from '@/constants';
 
-// TODO: should be one module
-function safeRoundTwoDecimals(input: string | number | undefined, defaultValue = '0'): string {
-  return safeRound(input, defaultValue, 2);
-}
-
-function safeRound(input: string | number | undefined, defaultValue: string, decimals: number) {
-  if (input === undefined) return defaultValue;
-  try {
-    const number = new Big(input);
-    return number.round(decimals).toString();
-  } catch {
-    return defaultValue;
-  }
-}
```

Comment:
I removed these methods as they turned out to be redundant in favor of `formatPercentage`.